### PR TITLE
Update docs for API ML extending websocket proxy

### PR DESCRIPTION
**Pull request by pablocarle**
### Your checklist for this pull request
🚨Please review the [guidelines for contributing](../docs/contribute/contributing.md) to this repository.

- [ ] If the changes in this PR is part of the next future release, make this pull request against the **docs-staging** branch which will be published at the next release boundary. If the changes in this PR are part of the current release, use the default base branch, **master**. For more information about branches, see https://github.com/zowe/docs-site/tree/master#understanding-the-doc-branches. 
      
- [ ] If this PR relates to GitHub issues in `docs-site` or other repositories, please list in Description, prefixed with **close**, **fix** or **resolve** keywords.

### Description (including links to related git issues)
Please describe your pull request.

:heart:Thank you!

### CODE REVIEW

File: docs/extend/extend-apiml/websocket.md [hunk 1]
```diff
@@ -15,39 +24,65 @@ The service can define what WebSocket services are exposed using Eureka metadata
                             gatewayUrl: "ws/v1"
                             serviceUrl: /discoverableclient/ws
 
-These metadata make it possible for requests from `wss://gatewayHost:port/serviceId/ws/v1/path` to map to `ws://serviceHost:port/discoverableclient/ws/path`,
-where:
+These metadata make it possible for requests from `wss://gatewayHost:gatewayPort/serviceId/ws/v1/path` to map to `wss://serviceHost:servicePort/discoverableclient/ws/path`, where:
```
[the remark this answers is not in the document]
Comment (pablocarle) [reply docs/extend/extend-apiml/websocket.md:27]: - I didn't change the original text, but I assume the `discoverableclient` part is there to highlight that it's the URL on the service side.
- I think discoverable client is not using this ws/v1 pattern currently, but I'll verify

File: docs/extend/extend-apiml/websocket.md [hunk 2]
```diff
@@ -15,39 +24,65 @@ The service can define what WebSocket services are exposed using Eureka metadata
...
+
 ```
 SERVER_WEBSOCKET_SUPPORTEDPROTOCOLS=v12.stomp,v11.stomp,wamp,soap
 ```
-## High availability
+
+### High availability
 
 In the high availability scenario, the API Gateway makes it possible to open a new Websocket session by utilizing the load balancing mechanism. 
 Communication between the client and the server is handled by the API Gateway by propagating the session to a live instance.
```
[the remark this answers is not in the document]
Comment (pablocarle) [reply docs/extend/extend-apiml/websocket.md:63]: I think it's the second option, I'll verify and update accordingly.

File: docs/extend/extend-apiml/websocket.md [hunk 3]
```diff
@@ -13,41 +22,66 @@ The service can define what WebSocket services are exposed using Eureka metadata
...
   
 * **`path`**  
   is the remaining path segment in the URL.
 
-## Security and Authentication
+## Architecture
+
+In WebSocket routing, the API ML Gateway acts as both a WebSocket server for the client requesting this connection, and as a WebSocket client.
+
+Client (i.e. web browser) <-> Gateway (WebSocket Server) - Gateway (WebSocket Client) <-> Service's WebSocket Server
```
Comment (janan07) [inline docs/extend/extend-apiml/websocket.md:39]: Perhaps we should add a line to say what this arrangement is. Something like, "The following schema describes the arrangement between client-side and server-side components where the Gateway serves as both the Websocket server as well as the Websocket client."... or something like that.

**Review by janan07 — APPROVED** [review]
Reviewed and edited by tech info

**Pull request merged**